test/scripts/jenkins_visual_regression.sh

```diff
@@ -11,8 +11,7 @@ mkdir -p "$installDir"
 tar -xzf "$linuxBuild" -C "$installDir" --strip=1
 
 echo " -> running visual regression tests from kibana directory"
-checks-reporter-with-killswitch "X-Pack visual regression tests" \
```

Comment:
When this job is actually run in a PR it fails because of the `-- --`, there's apparently too many or not enough escapes or something to pass through the `--` just to the `percy exec` call, which requires it... Since the visual regression stuff isn't currently run in PRs I just removed the checks reporter here.